.github/workflows/codeql.yml

```
@@ -13,86 +1,128 @@
+name: "CodeQL Advanced Scan"
 
 on:
   push:
     branches: [ "versions" ]
   pull_request:
     branches: [ "versions" ]
   schedule:
-    - cron: '31 8 * * 2'
+    - cron: '31 8 * * 2' # every Tuesday at 08:31 UTC
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)